pilot/pkg/bootstrap/server.go

```diff
@@ -87,6 +89,10 @@ var (
 		plugin.Health,
 		plugin.Mixer,
 	}
+
+	enableElection = env.RegisterBoolVar("MASTER_ELECTION",
```

Comment:
why not just disable if no kube client?

Reply:
We may want to disable even if kube client is present. 

You are right, we could also do an || kubeClient == nil